src/control/cmd/dmg/pool.go

```diff
@@ -86,9 +72,14 @@ type PoolCreateCmd struct {
 func (c *PoolCreateCmd) Execute(args []string) error {
 	msg := "SUCCEEDED: "
 
-	scmBytes, nvmeBytes, err := calcStorage(c.log, c.ScmSize, c.NVMeSize)
+	scmBytes, err := humanize.ParseBytes(c.ScmSize)
+	if err != nil {
+		return errors.Wrap(err, "pool SCM size")
+	}
+
+	nvmeBytes, err := humanize.ParseBytes(c.NVMeSize)
```

Comment:
I believe that `humanize.ParseBytes()` will return an error for empty strings, which means that the use case of only supplying `--scm-size` won't work anymore. Not sure if that was intentional.